src/vs/base/node/id.ts

```diff
@@ -81,30 +82,16 @@ export function getMachineId(): Promise<string> {
 		.then(id => id || uuid.generateUuid())); // fallback, generate a UUID
 }
 
-function getMacMachineId(): Promise<string> {
-	return new Promise<string>(resolve => {
-		Promise.all([import('crypto'), import('getmac')]).then(([crypto, getmac]) => {
-			try {
-				getmac.getMac((error, macAddress) => {
-					if (!error) {
-						resolve(crypto.createHash('sha256').update(macAddress, 'utf8').digest('hex'));
-					} else {
-						resolve(undefined);
-					}
-				});
-
-				// Timeout due to hang with reduced privileges #58392
-				// TODO@sbatten: Remove this when getmac is patched
-				setTimeout(() => {
-					resolve(undefined);
-				}, 10000);
-			} catch (err) {
-				errors.onUnexpectedError(err);
-				resolve(undefined);
-			}
-		}, err => {
+function getMacMachineId(): Promise<string | undefined> {
+	return import('crypto').then(crypto => {
+		return getMac().then((macAddress: string) => {
```

Comment:
We no longer seem to depend on `getmac`. Why does this PR not include the necessary changes to remove `getmac` from our `package.json`, `yarn.lock`, `getmac.d.ts,` etc.?